socorro/processor/processor_pipeline.py

```diff
@@ -136,11 +136,6 @@ class ProcessorPipeline(RequiredConfig):
         doc="absolute path to symbol cache",
         default=os.path.join(tempfile.gettempdir(), "symbols"),
     )
-    required_config.minidumpstackwalk.add_option(
```

Comment:
Instead of the minidump stackwalker having its own temporary directory (separate from the one the dumps are using), we use the temporary directory the processor app created for processing this crash report.